### ISSUE TITLE
util.c:188) ioctl(): SIOCGIFADDR No such device

### ISSUE THREAD

**Issue by FranklinYan** [wifidog/wifidog-gateway#267]
error occur while execute the wifidog server by type ". /system/bin/wifidog" in the terminal.

**Comment by hbl0307106015** [comment]
Please check your configuration file and verify that your gateway interface is set correctly.

**Comment by FranklinYan** [comment]
yes, you are right. i provide a error gateway interface. it's okay after modified it.

**Comment by henok400** [comment]
Remove 
#ExternalInterface eth0
or put front # then can solve error. 
tell me any other problem wifidog im from cloudwifizone
